packages/aws-cdk-lib/aws-lambda-nodejs/lib/types.ts

```diff
@@ -210,7 +210,7 @@ export interface BundlingOptions extends DockerRunOptions {
    *
    * For example, to add the [--log-limit](https://esbuild.github.io/api/#log-limit) flag:
    *
-   * ```text
+   * ```ts
```

Comment:
This requires the code block to be valid TypeScript, since it needs to be transpiled to other languages.

Reply:
changed 👍